pilot/pkg/proxy/envoy/v2/ads.go

```diff
@@ -165,7 +165,7 @@ func (s *DiscoveryServer) StreamAggregatedResources(stream ads.AggregatedDiscove
 	}
 
 	con := &XdsConnection{
-		pushChannel:   make(chan *XdsEvent, 1),
+		pushChannel:   make(chan *XdsEvent, 100),
```

Comment:
@costinm curious why 100? Would be good to define a constant with a comment explaining its value.

Reply:
Probably - the bug was not here but in pushing with a lock. Still added this as an extra failsafe - while throttling and event handling is reimplemented.